src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Minkowski.hpp

```diff
@@ -39,10 +39,11 @@ class Minkowski {
   using options = tmpl::list<>;
   static constexpr OptionString help{
       "Minkowski solution to Einstein's Equations"};
+  static constexpr size_t volume_dim = Dim;
 
   Minkowski() = default;
-  Minkowski(const Minkowski& /*rhs*/) = delete;
-  Minkowski& operator=(const Minkowski& /*rhs*/) = delete;
+  Minkowski(const Minkowski& /*rhs*/) = default;
```

Comment:
@geoffrey4444 - Why did you default these constructors? Do you want to be able to copy and not just move?